src/diffusers/pipeline_utils.py

```diff
@@ -541,7 +541,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             # if the model is in a pipeline module, then we load it from the pipeline
             if name in passed_class_obj:
                 # 1. check that passed_class_obj has correct parent class
-                if not is_pipeline_module:
+                if not is_pipeline_module and passed_class_obj[name] is not None:
```

Comment:
Since `safety_checker` is `OnnxRuntimeModel` and not a pipeline module like in the pytorch version, it produced an error further in this branch when `safety_checker==None`. 
So let's just skip anything that's `None` here.

Reply:
ok!